articles/azure/azs-how-to-obtain-clientid-via-api.md

```diff
@@ -58,17 +58,17 @@ The Authentication/Login Endpoint can be https:/\/login.microsoftonline.com/{ten
 
 **How to obtain Access Token Request Body and Authentication Information**
```

Comment:
For the Request Authentication Endpoint heading above, you need to remove the space after the first ** so that the heading formats correctly as bold text

Reply:
Adjusted all sub headings